flow/handler_test.go

```diff
@@ -57,8 +60,23 @@ func TestPostFlow_ShouldAddFlowToRepoForValidRequest(t *testing.T) {
 	assert.Equal(t, expectedFlow, actualFlow)
 }
 
-func TestPostFlow_ShouldReturn400ForInvalidRequest(t *testing.T) {
+func TestPostFlow_ShouldReturn500WhenFlowIsEmpty(t *testing.T) {
```

Comment:
Shouldn't there be a test with valid json (so to speak) but that does not satisfy our schema? Missing the name or steps or something? There might be tests that do some of this but the idea is that we use tests as documentation to the expected behaviour too

Reply:
Added another test 👍